test-all

```diff
@@ -2,6 +2,18 @@
 
 set -e
 
+# Make sure the Tensorflow version in the -sys build script matches the one in
+# the run-valgrind script.
+version_build_script=`grep "const VERSION" tensorflow-sys/build.rs | sed 's|.*"\([0-9\.]*\)";|\1|g'`
```

Comment:
sed tip: You can get rid of the grep and backslashes by doing

    version_build_script=`sed -En 's|const VERSION.*"([0-9.]*)";|\1|p' tensorflow-sys/build.rs`

 Doesn't really matter in this case (mostly just for processing huge amounts of data or avoiding pipelines), just thought I'd share. :)